testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -575,6 +575,13 @@ def test_invalid_smarts_sel_raises_error(self, u2):
         with pytest.raises(ValueError, match="not a valid SMARTS"):
             u2.select_atoms("smarts foo")
 
+    def test_passing_args_to_converter(self):
+        u = mda.Universe.from_smiles("O=C=O")
+        with pytest.raises(AttributeError, match="No hydrogen atom"):
```

Comment:
this should be two tests

Reply:
That first test was just me making sure I reproduce the same error as in the original issue, it's not really needed now so I'll delete it

Reply:
I _think_ this is the only thing that needs fixing, I guess it's just a case of deleting these two lines?